File: Core/SSE/SSEConnection.m
//
//  SSEConnection.m
//  CocoaHTTPServer
//
//  Created by Travis Bader on 5/6/16.
//  Copyright © 2016 Appareo Systems. All rights reserved.
//

#import "SSEConnection.h"
#import "SSEResponse.h"

@implementation SSEConnection

- (NSObject<HTTPResponse> *)httpResponseForMethod:(NSString*)method URI:(NSString*)path {
    return [SSEResponse new];
}

@end

File: Samples/ServerSentEventsExample/SSEConnection.swift
//
//  SSEConnection.swift
//  ServerSentEventsExample
//
//  Created by Appareo on 2/10/16.
//
//

import Foundation

public class SSEConnection: HTTPConnection {

    //- (NSObject<HTTPResponse> *)httpResponseForMethod:(NSString *)method URI:(NSString *)path;
    public func httpResponseForMethod(method:String, path:String) -> HTTPResponse {
        let response = HTTPResponseTest()
        return response
    }
    
}
